pom.xml

```diff
@@ -90,44 +90,25 @@
             <version>3.1.1</version>
         </dependency>
         <dependency>
-             <groupId>org.springdoc</groupId>
-             <artifactId>springdoc-openapi-ui</artifactId>
-             <version>1.7.0</version>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-ui</artifactId>
+            <version>1.7.0</version>
         </dependency>
         <dependency>
             <groupId>org.glassfish</groupId>
             <artifactId>javax.el</artifactId>
             <version>3.0.1-b12</version>
         </dependency>
-        <dependency>
-            <groupId>io.jsonwebtoken</groupId>
-            <artifactId>jjwt-api</artifactId>
-            <version>0.11.5</version>
-        </dependency>
         <dependency>
             <groupId>io.jsonwebtoken</groupId>
             <artifactId>jjwt-impl</artifactId>
             <version>0.11.5</version>
-            <scope>runtime</scope>
-        </dependency>
-        <dependency>
-            <groupId>io.jsonwebtoken</groupId>
-            <artifactId>jjwt-jackson</artifactId>
-            <version>0.11.5</version>
-            <scope>runtime</scope>
```

Comment:
Looks like both jjwt-api and jjwt-jackson are needed and used in `src/main/java/org/isf/security/jwt/TokenProvider.java`